contribs/entropy2mnemonic/main.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"bufio"
+	"crypto/sha256"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/gnolang/gno/tm2/pkg/crypto/bip39"
+)
+
+const (
+	// MinEntropyLength is the minimum recommended entropy length for 160-bit security
+	MinEntropyLength = 27
+
+	// RecommendedEntropyLength is the recommended entropy length for good security
+	RecommendedEntropyLength = 43
+)
+
+func main() {
+	if err := run(os.Stdin, os.Stdout, os.Stderr, os.Args[1:]); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func run(stdin io.Reader, stdout, stderr io.Writer, args []string) error {
+	fs := flag.NewFlagSet("entropy2mnemonic", flag.ContinueOnError)
+	fs.SetOutput(stderr)
+
+	quiet := fs.Bool("quiet", false, "only output the mnemonic")
```

Comment:
In addition, as mentioned in the main README, and as per github/jaekwon/openpgp, a way to enforce a flag that masks the entropy input from stdin, as '*****', as passwords.

Reply:
[`ab1d466` (#4586)](https://github.com/gnolang/gno/pull/4586/commits/ab1d4663f86ad80ad583e9b2969ab4c523701b49)